packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -332,11 +289,33 @@ def request_tendermint_info(self) -> Generator[None, None, bool]:
             self.collection_complete = True
         return self.collection_complete
 
+    def format_genesis_data(
+        self,
+        collected_agent_info: Dict[str, Any],
+    ) -> Dict[str, Any]:
+        """Format collected agent info for genesis update"""
+
+        validators = []
+        for i, validator_config in enumerate(collected_agent_info.values()):
+            validator = dict(
+                address=validator_config["address"],
+                pub_key=validator_config["pub_key"],
+                power=self.params.voting_power,
```

Comment:
Shouldn't this be the voting power of the validator node, eg something like `power=validator_config["power"]`

Reply:
@Karrenbelt could you explain this choice?

Reply:
so this was "DEFAULT_VOTING_POWER" when I wrote it, for a reason. That is because the validator config and public key are requested and shared among agents, but we cannot rely on parties reporting their own voting power honestly. The default voting power on Tendermint is `10`, hence we don't want to set it to `1` either and somehow risk those not updating to have more voting power, although technically they cannot (should not be able to) connect to the other nodes.

Either way, for out application it would be good to have voting power set to default all equal (for now) and what we could also consider is to have the agents reach consensus on the tendermint configuration. Quite frankly I don't know what happens if there are inconsistencies in e.g. voting power that different agents have assigned to other agents in their tendermint config. Anyone else?

Reply:
Yes, that makes sense. Therefore I'd suggest that I remove it from the config files and hardcode it again.

Reply:
the params we have set, are part of the IPFS hash as well right, so if you specify voting power in params or in the file as a constant should make no difference